workflows/bacterial_genomics/amr_gene_detection/README.md

```diff
@@ -0,0 +1,30 @@
+# AMR gene detection in an assembled bacterial genome subworkflow (v1.0)
+
+This subworkflow use assembled bacterial genome fasta files and executes the following steps:
```

Comment:
Not sure I would call it a subworkflow since it doesn't make any assumptions about how the bacterial consensus sequence was generated -> would work on any fasta genome you provide, wouldn't it?

Reply:
I agree with you, so I changed it to "workflow" and clarified that it is possible to use any fasta file :smile: